.github/workflows/enso4igv.yml

```diff
@@ -10,7 +10,65 @@ on:
       - "tools/enso4igv/**/*"
 
 jobs:
-  build:
+  build_linux_parser:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install rustup
+        run: |
+          rustup target add x86_64-unknown-linux-musl
+
+      - name: Build Enso Parser
+        working-directory: .
+        run: |
+          RUSTFLAGS="-C target-feature=-crt-static" cargo build -p enso-parser-jni -Z unstable-options --target x86_64-unknown-linux-musl --out-dir target/lib/linux/amd64/
+
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: linux_parser
+          path: |
+            target/lib/**
+
+  build_mac_parser:
+    runs-on: macos-12
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build Enso Parser
+        working-directory: .
+        run: |
+          RUSTFLAGS="-C target-feature=-crt-static" cargo build -p enso-parser-jni -Z unstable-options --out-dir target/lib/mac/amd64/
+
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: mac_parser
+          path: |
+            target/lib/**
+
+  build_windows_parser:
+    runs-on: windows-2019
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build Enso Parser
+        working-directory: .
+        run: |
+          RUSTFLAGS="-C target-feature=-crt-static" cargo build -p enso-parser-jni -Z unstable-options --out-dir target/lib/windows/amd64/
```

Comment:
structure:

>    lib: contains native JNI libraries and basic boot files. The naming policy of the libraries is in hands of the modules, as that it has worked well enough till now. If it proves unsufficient following conventions shall be obeyed: There are subdirectories under the lib directory that reflect the architecture (e.g. i386, obtained via System.getProperty ("os.arch")) and possibly also subdirectory identifying the system (e.g. i386/linux, obtained via System.getProperty ("os.name").toLowerCase ()). When the system is looking for a particular library it scans the deepest directories first and then the shallow ones (lib/i386/linux, lib/i386, lib). 


can you please port here what are the values of `os.name` and `os.arch`  when executed on Windows and (new) Mac?